packages/data/src/useWalkthroughTestData.ts

```diff
@@ -70,6 +73,26 @@ export function getMultiChoiceMultipleQuestion() {
   };
 }
 
+export function getNumberFloatQuestion() {
+  // find the first question that is a number float question
+  const questions: {
+    [key: string]: QuestionDisplayData | QuestionVariableData;
+  } = testCase91014.questions;
+  const questionKey = Object.keys(questions).find((questionId) => {
+    const question = questions[questionId] as QuestionDisplayData;
+    return question.walkthroughItemType === WalkthroughItemTypeNumberFloat;
+  });
+
+  if (!questionKey || !questions[questionKey]) {
+    throw new Error("No numberFloat question info");
```

Comment:
I think crash reporting might be helpful to log errors such as this. Crashlytics?